### DISCUSSION TITLE
Transferring Temperature data over Time in CSV Using Piecewise Multilinear

### DISCUSSION THREAD

**Discussion by tt1me8** [idaholab/moose#28121] [Q&A Modules: General]
Trying to transfer temperature data for a surface over to MOOSE using Piecewise Multilinear. What values do I use for the nodes for which I do not have temperature data when I'm stepping through the gridded data that I need to write into the data file?

**Comment by GiudGiud** [comment]
Hello

Whichever value you choose to use you ll have to be careful not to use this part of the domain in your simulation.

If would recommend using a "nearest-valid-value" approach, in which the linear interpolation becomes a constant outside of the known domain

Guillaume

**Reply by tt1me8** [reply]
Hmm, so there's no null value that I can throw in there as a placeholder then?

**Reply by GiudGiud** [reply]
You can try putting a very big number there. I m sure we have some support for NaN too but for this kind of custom behavior you should code your own object
